### DISCUSSION TITLE
Pre-Configure Git LFS Config in Backend

### DISCUSSION THREAD

**Discussion by BenAigan** [git-lfs/git-lfs#4540] [General]
Hi,

I would like to handle the config for setting up my users with Git LFS with Artifactory as the back end storage so that it works seamlessly.  I want to take all the config out of users hands to avoid the usual overheads of user's being users.

I have tried adding the lfs.url to the Bitbucket backend repository config file on the bare repo at creation time but this does not come through when cloning the repo.  I have tried adding an include in the same way that the system gitconfig is loaded but still no luck.

What are your thoughts on this?

**Comment by bk2204** [comment]
Hey,

I don't know how Bitbucket operates, so it's hard to say how to configure it, but my suggestion is to have some sort of template script to set up a new repository.  You're almost certainly going to want to set `lfs.url` in the `.lfsconfig` in the repository, and the only way to do that effectively is to have a script to set up the new repositories and commit that file as part of it.  It's possible that Bitbucket has some sort of configuration option to do that for you using a template mechanism, but if so, I'm not familiar with it.

**Reply by ShahriarSumon07** [reply]
> Hey,
> 
> I don't know how Bitbucket operates, so it's hard to say how to configure it, but my suggestion is to have some sort of template script to set up a new repository. You're almost certainly going to want to set `lfs.url` in the `.lfsconfig` in the repository, and the only way to do that effectively is to have a script to set up the new repositories and commit that file as part of it. It's possible that Bitbucket has some sort of configuration option to do that for you using a template mechanism, but if so, I'm not familiar with it.
